example/index.jsx

```diff
@@ -5,6 +5,7 @@ import ReactDOM from 'react-dom'
 import classNames from 'classnames'
 import { PrimaryTitle } from '../components/Text'
 import * as examples from './examples'
+import 'normalize.css'
```

Comment:
Wouldn't you want to put this below the babel-polyfill ?
I am unsure if the CSS for PrimaryTitle or examples are added before normalize.css?